prodbox.Dockerfile

```diff
@@ -20,30 +20,25 @@ ENV LD_LIBRARY_PATH=/usr/local/lib
 # Set the working directory to /dust
 WORKDIR /dust
 
+COPY . .
```

Comment:
This is to get the `.git` folder

Reply:
Can you copy just the .git, at the very last step of the dockerfile ? Otherwise I think it's going to break cache all the time

Reply:
Agree that this is going to affect caching if we COPY . . at the beginning, but if we only copy .git and not all the folders and files, pulling new code is going to be impossible no?
The upside here is that we won't have to deploy prodbox as often anymore, and we can even decide to deploy it every night to make sure we are never too far from the latest version, pull  new code during the day

Reply:
> if we only copy .git and not all the folders and files, pulling new code is going to be impossible no?

:/ this is going to be quite slow every time we actually have to deploy I think... but we probably have to if we want to have git indeed.

Did you test that the build still works with copy . . ?

Reply:
@fontanierh it passed all the subsequent instructions of this file so I suppose it does.